runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationConfig.java

```diff
@@ -64,6 +64,16 @@ default boolean showSeeds()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "showSacks",
```

Comment:
Fix your indentation (continuation indent 8>4)

Reply:
For some reason IntelliJ defaults to this indenting when starting a new line. Visual Studio and VS Code have spoiled me too much. Either way, fixed.

Reply:
Yeah, I know it defaults to that. We have wiki page here with instructions on how to import RL code style to intellij: https://github.com/runelite/runelite/wiki